predictors_reliability.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<p>This looks good. I like the this plot where we can check how may principal components we need</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/santiviquez/lie-detection/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>